notebooks/community/model_evaluation/automl_tabular_classification_model_evaluation.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #80.            problem_type=prediction_type,
</pre><p>Problem type is not required since classification_metrics is provided.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/911/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='59'/>

Reply:
<p>removed.</p>